src/halse_se05x.c

```diff
@@ -93,6 +93,12 @@ enum cmd_error {
 };
 #define CMD_ERROR_MASK 0x03
 
+static const uint8_t noreset_atr[] = {
```

Comment:
Please add a comment, that this is actually an observed ATR, which is reported when using a real device.